test/python/test_graph.py

```diff
@@ -16,31 +16,54 @@
 import unittest
 import torch
 
-from graphlearn_torch.data import CSRTopo, Graph
+from graphlearn_torch.data import Topology, Graph
 
 
 class GraphTestCase(unittest.TestCase):
   def setUp(self):
-    self.indptr = torch.tensor([0, 2, 4, 7, 8], dtype=torch.int64)
-    self.indices = torch.tensor([0, 1, 1, 3, 2, 3, 4, 5], dtype=torch.int64)
-    self.edge_ids = torch.tensor([0, 1, 2, 3, 4, 5, 6, 7], dtype=torch.int64)
-    self.csr_topo = CSRTopo(
-      edge_index=(self.indptr, self.indices),
-      edge_ids=self.edge_ids,
+    """
+    input graph:
+      r/c	0	1	2	3	4	5
```

Comment:
The table is not correct from the second line. Should shift to the right by one space.

Reply:
yes